torchrl/modules/tensordict_module/rnn.py

```diff
@@ -399,3 +400,379 @@ def _lstm(
+    - Optional multi-step execution: with torch.nn, one has to choose between
+      :class:`torch.nn.GRUCell` and :class:`torch.nn.GRU`, the former being
+      compatible with single step inputs and the latter being compatible with
+      multi-step. This class enables both usages.
+
+
+    After construction, the module is *not* set in temporal mode, ie. it will
+    expect single steps inputs.
+
+    If in temporal mode, it is expected that the last dimension of the tensordict
+    marks the number of steps. There is no constrain on the dimensionality of the
+    tensordict (except that it must be greater than one for temporal inputs).
+
+    Args:
+        input_size: The number of expected features in the input `x`
+        hidden_size: The number of features in the hidden state `h`
+        num_layers: Number of recurrent layers. E.g., setting ``num_layers=2``
+            would mean stacking two GRUs together to form a `stacked GRU`,
+            with the second GRU taking in outputs of the first GRU and
+            computing the final results. Default: 1
+        bias: If ``False``, then the layer does not use bias weights.
+            Default: ``True``
+        dropout: If non-zero, introduces a `Dropout` layer on the outputs of each
+            GRU layer except the last layer, with dropout probability equal to
+            :attr:`dropout`. Default: 0
+        proj_size: If ``> 0``, will use GRU with projections of corresponding size. Default: 0
+
+    Keyword Args:
+        in_key (str or tuple of str): the input key of the module. Exclusive use
+            with ``in_keys``. If provided, the recurrent keys are assumed to be
+            ["recurrent_state"] and the ``in_key`` will be
+            appended before this.
+        in_keys (list of str): a pair of strings corresponding to the input value and recurrent entry.
+            Exclusive with ``in_key``.
+        out_key (str or tuple of str): the output key of the module. Exclusive use
+            with ``out_keys``. If provided, the recurrent keys are assumed to be
+            [("recurrent_state")] and the ``out_key`` will be
+            appended before these.
+        out_keys (list of str): a pair of strings corresponding to the output value,
+            first and second hidden key.
+            .. note::
+              For a better integration with TorchRL's environments, the best naming
+              for the output hidden key is ``("next", <custom_key>)``, such
+              that the hidden values are passed from step to step during a rollout.
+        device (torch.device or compatible): the device of the module.
+        gru (torch.nn.GRU, optional): a GRU instance to be wrapped.
+            Exclusive with other nn.GRU arguments.
+
+    Attributes:
+        temporal_mode: Returns the temporal mode of the module.
+
+    Methods:
+        set_recurrent_mode: controls whether the module should be executed in
+            temporal mode.
```

Comment:
the attribute is `temporal_mode` but the method is `set_recurrent_mode` is it possible to unify naming?